src/agentscope/agents/agent.py

```diff
@@ -248,3 +248,39 @@ def to_dist(
             lazy_launch=lazy_launch,
             agent_id=self.agent_id,
         )
+    
```

Comment:
Create a new agent class in the example folder, not to change the agentbase here